vhost-device-sound/src/audio_backends/pipewire.rs

```diff
@@ -375,18 +375,16 @@ impl AudioBackend for PwBackend {
 
                                 let mut start = 0;
                                 while n_samples > 0 {
```

Comment:
I think we can remove this. We shall only write a request, i.e., period_bytes but not more than that.